lib/goliath/runner.rb

```diff
@@ -299,7 +299,8 @@ def setup_stdout_logger(log, log_format)
      def run_server
        log = setup_logger
 
-       log.info("Starting server on #{@address}:#{@port} in #{Goliath.env} mode. Watch out for stones.")
+       ssl = [nil, 'off'].include?(@server_options[:ssl]) ? false : true
```

Comment:
I think our OptionsParser might be misleading you a bit.. server_options[:ssl] is actually treated as a binary setting. You can see the runtime check here: https://github.com/postrank-labs/goliath/blob/master/lib/goliath/server.rb#L88

As such, I think you remove the include? line and just do the binary check in the log.info.